packages/style-kit/src/_mixins.scss

```
@@ -3,11 +3,13 @@
 @import './typography';
 @import './cards';
 @import './tags';
+@import './controls';
 
 @mixin create-meeco-theme($theme-config) {
   @include buttons-main($theme-config);
   @include inputs-main($theme-config);
   @include typography-main($theme-config);
   @include cards-main($theme-config);
   @include tags-main($theme-config);
+  @include controls-main;
```

Failure is here - needs `($theme-config)`

```suggestion
  @include controls-main($theme-config);
```